backend/czi_hosted/test/performance/scale_test_annotations.py

```diff
@@ -7,10 +7,10 @@
 """
 To run this script sign into cellxgene in the desired environment and grab the returned cookie, update the cookie
 variable below with your cookie and run the following command to see results in the terminal:
-locust -f server/test/performance/scale_test_annotations.py --headless -u 30 -r 10 --host https://api.cellxgene.dev.single-cell.czi.technology/cellxgene/e/ --run-time 5m 2>&1 | tee locust_dev_stats.txt
+locust -f czi_hosted/test/performance/scale_test_annotations.py --headless -u 30 -r 10 --host https://api.cellxgene.dev.single-cell.czi.technology/cellxgene/e/ --run-time 5m 2>&1 | tee locust_dev_stats.txt
```

Comment:
shouldn't this be `backend/czi_hosted/...`?   (ditto for the same string on line 13)